added vagrantbox

rgeraads opened the pull request.

rgeraads committed:
added vagrantbox

mbed67 commented on ansible/playbook.yml:
```diff
@@ -0,0 +1,51 @@
+  - name: run database migrations
+    shell: cd /vagrant && php artisan migrate
+
+  - name: download composer
```
When I run this, ansible gives a warning: `[WARNING]: Consider using get_url module rather than running curl`. Could you please use get_url instead?

rgeraads replied:
tried to look into this one, but failed so far. I'll figure out something eventually.

mbed67 commented on ansible/playbook.yml:
```diff
@@ -0,0 +1,51 @@
+    shell: curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/local/bin --filename=composer
+
+  - name: composer install
+    shell: composer install
```
It cannot find the composer.json, because it looks in /home/vagrant but the project lives in /vagrant.
Could you fix this like you did with the migrations?

rgeraads committed:
improved ansible provisioning

rgeraads commented:
added some improvements
- moved npm prune & npm install from composer.json to ansible provisioning because it was hitting the 300sec timeout.
  - added nginx (still figuring out correct config)
    added write access to storage dir

this guy helped me out a bit
![img_1620](https://cloud.githubusercontent.com/assets/6044164/17645360/e7c76c38-61a3-11e6-89b3-1ed02c8b4592.JPG)

mbed67 commented:
Ahhh prachtig! Gefeliciteerd!!   :-D
En dank je voor de aanpassingen. Morgen wordt een drukke dag hier i.v.m. feestje van mijn dochter en voorbereidingen. Dus wss kijk ik daarna.

Groetjes, en geniet van je zoon!
Marieke
Sent from my FairPhone

Op 13 aug. 2016 10:19 PM schreef Randy Geraads notifications@github.com:

added some improvements
-   moved npm prune & npm install from composer.json to ansible provisioning because it was hitting the 300sec timeout.
  -   added nginx (still figuring out correct config) added write access to storage dir

this guy helped me out a bit
[img_1620]https://cloud.githubusercontent.com/assets/6044164/17645360/e7c76c38-61a3-11e6-89b3-1ed02c8b4592.JPG

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHubhttps://github.com/mbed67/sanghaplanner/pull/2#issuecomment-239639582, or mute the threadhttps://github.com/notifications/unsubscribe-auth/AGzybTZyNE_Wh2KttOJUJzYnK8xUJEHjks5qfia4gaJpZM4JihG3.

mbed67 commented on ansible/playbook.yml:
```diff
@@ -0,0 +1,72 @@
+#    get_url:
+#      url: https://getcomposer.org/installer
+#      dest: /usr/local/bin/composer
+#      mode: 0755
```
This works, but you get a file with the content of composer-setup.php at the destination.
You could use a dest of **/tmp/composer-setup.php** and then make another task which installs it to be /usr/local/bin/composer

rgeraads committed:
updated composer installation
